discovery_child_development/getters/labels.py

```diff
@@ -1,9 +1,9 @@
 """Module for getting labelled data from S3"""
 import pandas as pd
 from pathlib import Path
-from discovery_child_development import S3_BUCKET, PROJECT_DIR
+from discovery_child_development import S3_BUCKET, PROJECT_DIR, get_yaml_config
```

Comment:
If you wanted, you could replace `get_yaml_config` with `config` and remove the config= in the function. As that is already set up in the __init__ file.

Reply:
Hmm, but I'm loading in a different config file from the global config - is that what you meant?

Reply:
Oh yeah I see! Would it not be simpler to set it up in the init file as well? Although I suppose it doesn't matter. Is there a reason it is in the pipeline rather than the config folder?

Reply:
Thanks Jess, I think both ways would be OK. I agree that having multiple configs makes things more complicated - I was trying to compartmentalise the labelling scripts so that everything relevant to a labelling routine is in one folder. I think let's keep it as is, but I'll make a not to consider combining all the configs.